src/main/thrift/parquet.thrift

```diff
@@ -237,6 +237,97 @@ struct SizeStatistics {
    3: optional list<i64> definition_level_histogram;
 }
 
+/**
+ * Interpretation for edges of GEOMETRY logical type, i.e. whether the edge
+ * between points represent a straight cartesian line or the shortest line on
+ * the sphere. It applies to all non-point geometry objects.
+ */
+enum Edges {
```

Comment:
Nit: can we avoid the plural? Perhaps make it `EdgeShape` or `EdgeKind`.

Reply:
We use the same definition from GeoParquet (https://github.com/opengeospatial/geoparquet/blob/main/format-specs/geoparquet.md) and we try to be consistent with GeoParquet.

Reply:
Pity. If you want to store a collection of `Edges`, how do you name it?

Reply:
That said, I understand it's probably not a practical concern :-)

Reply:
emm.. Why would someone want to store a collection of edges? For each geometry column, the `Edges` field has only 1 possible value.

Reply:
I'd say `EdgeInterpolation` is the better name than `EdgeKind`. +1 for keeping the attribute of the type the same as GeoParquet for consistency and simplicity, but makes sense to me that the enum name could be more descriptive.

Reply:
I find the term `EdgeInterpolation` quite misleading. What we seem to want to convey here is how the edges between points are interpreted. How they are interpreted has to do with the underlying space in which the geometry is embedded. For example for WGS84 the edges are by definition spheroidal edges (not spherical, which is why I have made other comments about the use of the word `SPHERICAL`). If you have SRID 3857, then it is a planar coordinate system. I guess the word "interpolation" could mean what you use to _interpolate_ between vertices, but this feels like an awkward use of the word.

Reply:
@mkaravel can you suggest a better name here, or can we fall back to a generic name like `Edges` and put the explanation in the comments, which will leave the door open in the future without breaking the existing implementation.

Reply:
It is true that the "edge interpolation" depends on the underlying space, but not only. For example on a plane, we can connect two points by the shortest path (a straight line), or we can try to keep the first derivatives continuous (splines). Likewise on a sphere, we can connect with the shorted paths (geodesic lines), or try to keep the headings continuous at each point.

So it is a little bit more than only the underlying space. If we interpret "interpolation" as "the mathematical formulas that we choose to use for computing intermediate points", than "edge interpolation" seems appropriate to me.